src/components/subscription-choices/subscription-choices.scss

```diff
@@ -0,0 +1,90 @@
+@use '../../compiled/tokens/scss/breakpoint';
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/size';
+@use '../../mixins/a11y';
+@use '../../mixins/theme';
+
+///
+/// Theme properties
+///
+
+// Light theme
+@include theme.props {
+  --theme-color-background-subscription-form: #{color.$base-white};
+}
+
+// Dark theme
+@include theme.props(dark) {
+  --theme-color-background-subscription-form: #{color.$brand-primary};
+}
+
+// Alternate theme
+@include theme.props(alternate) {
+  --theme-color-background-subscription-form: #{color.$base-gray-lighter};
+}
```

Comment:
These are not necessary if you use the existing theme properties instead of setting new ones (see my other comment further down).

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/1679/commits/1362e84a6c29b9dc1c0bdaf05670702503bcf8f8